app/models/federal_register_client.rb

```diff
@@ -72,6 +72,7 @@ def a_sorn_title?(title)
 
   def sorn_params(result)
     {
+      agency_names: result.agency_names.join('<span class="agency-separator">|</span>'),
```

Comment:
Let's look into ways to apply the span at the view rather than in the db

Reply:
Done.